app/src/main/res/drawable/background_border.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#FFFFFF" />
+    <stroke android:width="1dp" android:color="@android:color/darker_gray" />
+    <padding android:left="0dp" android:top="0dp" android:right="1dp" android:bottom="0dp" />
+</shape>
```

Comment:
This is ugly, there must be a better workaround for the missing `elevation `in pre 21.

A quick search resulted in this: https://stackoverflow.com/questions/27385666/unknown-attribute-androidelevation